client/app/visualizations/pivot/index.js

```diff
@@ -21,7 +22,19 @@ function pivotTableRenderer() {
         if ($scope.queryResult.getData() !== null) {
           // We need to give the pivot table its own copy of the data, because it changes
           // it which interferes with other visualizations.
-          data = angular.copy($scope.queryResult.getRawData());
+          data = [];
+          const columns = $scope.queryResult.getColumns();
+          const rows = $scope.queryResult.getData();
+          rows.forEach((row) => {
+            const rowObj = {};
+            columns.forEach((col) => {
+              const colName = getColumnCleanName(col.name);
+              const value = formatValue($filter, clientConfig, row[col.name], col.type);
```

Comment:
Yes they do. Example is [here](http://nicolas.kruchten.com/pivottable/examples/mps_agg.html).
I have to come up with how to let people define it in pivottable editor, but currently have no idea.

I'm perfectly fine with dividing this PR into two, and making this only about the filters.
I assume what you mean is just replacing `.getRawData()` to `.getData()`?